Community-Supported/flights-data-incremental-refresh/README.md

```diff
@@ -0,0 +1,48 @@
+# flights-data-incremental-refresh
+## __Incremental Refresh using the OpenSkyApi__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+This sample is based on the content the Hyper team presented in the Hands on Training session "Hands-on: Leverage the Hyper Update API and Hyper API to Keep Your Data Fresh on Tableau Server" at Tableau Conference 2022 ([slides available here](https://mkt.tableau.com/tc22/sessions/live/430-HOT-D1_Hands-onLeverageTheHyperUpdate.pdf)).
+
+It demonstrates how to implement an incremental refresh based on the Hyper API and the [Hyper Update API](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_how_to_update_data_to_hyper.htm). It showcases this based on fligths data from the [OpenSkyAPI](https://github.com/openskynetwork/opensky-api). The sample should serve as a starting point for anyone looking to incrementally publish and update datasources on Tableau Server/Cloud. 
```

Comment:
add a description of what the script actually does, i.e. which scenario it implements.
I just had to read the script and reverse-engineer the intended logic from it. That shouldn't be necessary...